src/bn_txns.erl

```diff
@@ -45,7 +50,29 @@ init(Args) ->
             init(Args);
         {ok, State} ->
             persistent_term:put(?MODULE, State),
-            {ok, State#state{}}
+            case application:get_env(blockchain, store_historic_balances, false) of
```

Comment:
You wondered this already in the PR comment, and I agree.. putting account tracking in bn_txns does not feel right. Create a different module called bn_accounts.erl if it's not already there and track account balances there cleanly

Reply:
ok will do. there's already a bn_accounts.erl but it's not will it need to be a follower module. if i go the commit hook route I'm guessing it won't need to be a follower?

Reply:
I think I will copy the bn_wallets.erl format. Let me know if this is the wrong direction

Reply:
You may still need a follower to tie the commit hook callback with a block event / height happening. See what ETL does there.. A follower is not expensive to have btw